### PR TITLE
Namespace Worker Command

### LINKED ISSUE #73: Command Name Discussion
Looking over this plugin I'm wondering if the queue commands should be namespaced. Something like `bin/cake queue work` or `bin/cake queue worker` may allow for future improvements if we wanted to add something like `bin/cake queue flush <queue_name>`.

This would also have the benefit of being easier to identify when looking at processes on a system. As a system administrator, seeing `bin/cake worker` implies that it is a cake process, but is not helpful in relaying what is being worked. Is it some kind of file watcher worker or a PDF generator worker? Seeing `bin/cake queue work` would give clarity in a system process list.

### PR DESCRIPTION
**Pull request by amayer5125**
- Updated docs
- Updated tests

Closes #73

### CODE REVIEW

File: src/Plugin.php [hunk 1]
```diff
@@ -64,6 +64,8 @@ public function bootstrap(PluginApplicationInterface $app): void
      */
     public function console(CommandCollection $commands): CommandCollection
     {
-        return $commands->add('worker', WorkerCommand::class);
+        return $commands
+            ->add('queue work', WorkerCommand::class)
+            ->add('worker', WorkerCommand::class);
```
Comment (othercorey) [inline src/Plugin.php:69]: @markstory Should we just stick with the new convention and remove this in 1.0?

File: src/Plugin.php [hunk 2]
```diff
@@ -64,6 +64,8 @@ public function bootstrap(PluginApplicationInterface $app): void
      */
     public function console(CommandCollection $commands): CommandCollection
     {
-        return $commands->add('worker', WorkerCommand::class);
+        return $commands
+            ->add('queue work', WorkerCommand::class)
```
Comment (ADmad) [inline src/Plugin.php:68]: Don't really like the `work` vs `Worker` naming mismatch.

Comment (amayer5125) [reply src/Plugin.php]: Updated to `queue worker`.

File: src/Plugin.php (hunk 1 above)
Comment (markstory) [reply src/Plugin.php:69]: I'm fine with that. Aliases are low cost to maintain until we get to 1.0.

**Pull request merged**